src/elements/toniq-checkbox/toniq-checkbox.element.ts

```diff
@@ -35,7 +35,7 @@ export const ToniqCheckbox = defineToniqElement({
             transition: inherit;
             display: inline-flex;
             flex-grow: 1;
-            align-items: center;
+            align-items: start;
```

Comment:
This PR has commits mixed from #44. Please remove those commits to keep this PR clean and single-purpose.